pkg/clients/clients.go

```diff
@@ -1,14 +1,17 @@
 package clients
 
 import (
-	"github.com/coreos/etcd/client"
-	"k8s.io/client-go/rest"
-	"k8s.io/kubernetes/pkg/client/clientset_generated/clientset"
+	etcd "github.com/coreos/etcd/client"
+	k8s "k8s.io/kubernetes/pkg/client/clientset_generated/clientset"
+	"sync"
```

Comment:
@jmrodri if only I had some kind if Integrated Development Environment to help me with these mundane tasks...